src/components/drawer/drawer.d.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export interface DrawerPropTypes {
+  children: React.FunctionComponent | React.ComponentClass;
+  defaultExpanded?: boolean;
+  expanded?: boolean;
+  onChange?: () => void;
+  sidebar?: React.ReactNode;
+  expandedSettings?: string;
+  animationSpeed?: number;
```

Comment:
```suggestion
  animationSpeed?: string;
```

This was changed to a string e.g. `0.4s`.

Reply:
updated to string